### PR TITLE
omitted extra serialer for lineitems

### LINKED ISSUE #24: Bug: Cart resource has duplicate line items
When a client requests the `/profile/cart` resource, the following two keys are on the response object.

1. `lineitems`
1. `line_items`

The `line_items` key needs to be removed from the response.

### PR DESCRIPTION
**Pull request by jaclynmariefrench**
Description of PR that completes issue here...

## Changes

- took off comment on retrieve in the `lineitem.py`
- Omitted additional serializer on cart in `profile.py` view, there is already a serializer set in the `lineitem.py`

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET `http://localhost:8000/profile/cart` Gets cart items

**Response**

HTTP/1.1 200 OK

```json
{
    "id": 2,
    "url": "http://localhost:8000/orders/2",
    "created_date": "2019-04-12",
    "payment_type": null,
    "customer": "http://localhost:8000/customers/7",
    "lineitems": [
        {
            "id": 4,
            "product": {
                "id": 52,
                "name": "900",
                "price": 1296.98,
                "number_sold": 0,
                "description": "1987 Saab",
                "quantity": 2,
                "created_date": "2019-03-19",
                "location": "Vratsa",
                "image_path": null,
                "average_rating": null,
                "category_id": 2
            }
        },
        {
            "id": 5,
            "product": {
                "id": 33,
                "name": "Stratus",
                "price": 1199.91,
                "number_sold": 0,
                "description": "2001 Dodge",
                "quantity": 1,
                "created_date": "2019-04-06",
                "location": "Tianning",
                "image_path": null,
                "average_rating": null,
                "category_id": 2
            }
        },
        {
            "id": 6,
            "product": {
                "id": 71,
                "name": "Sebring",
                "price": 1045.66,
                "number_sold": 0,
                "description": "1999 Chrysler",
                "quantity": 4,
                "created_date": "2019-05-18",
                "location": "Namibe",
                "image_path": null,
                "average_rating": null,
                "category_id": 2
            }
        },
        {
            "id": 10,
            "product": {
                "id": 88,
                "name": "Element",
                "price": 1727.41,
                "number_sold": 0,
                "description": "2003 Honda",
                "quantity": 3,
                "created_date": "2019-05-28",
                "location": "Dukoh",
                "image_path": null,
                "average_rating": null,
                "category_id": 2
            }
        }
    ]
}
```

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database
- [ ] check and see if changes mentioned above are reflected in the code
- [ ] open up postman
- [ ] do GET `http://localhost:8000/profile/cart` and verify that the response is similar to the response code example

## Related Issues

- Fixes #24

### CODE REVIEW

File: bangazonapi/views/lineitem.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def retrieve(self, request, pk=None):
             HTTP/1.1 204 No Content
         """
         try:
-            # line_item = OrderProduct.objects.get(pk=pk)
+            line_item = OrderProduct.objects.get(pk=pk)
```
Comment (heymonicakay) [inline bangazonapi/views/lineitem.py]: This line of code should be deleted. The query below on line 54 gets the same data from the database with an added layer of filtering to ensure we only get back data that belongs to the authenticated user.

File: bangazonapi/views/profile.py [hunk 2]
```diff
@@ -178,14 +178,10 @@ def cart(self, request):
...
                     customer=current_user, payment_type=None)
                 line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False,
                                                 context={'request': request}).data
-                cart["order"]["line_items"] = line_items.data
-                cart["order"]["size"] = len(line_items.data)
```
Comment (heymonicakay) [inline bangazonapi/views/profile.py:188]: We do want to see the `size` property that shows how many line items are on the order. Look at the `count()` method, and see how you can apply it here to accomplish this.
NOTE: You will _not_ need to replace any of the other lines of code you removed

**Review by heymonicakay — CHANGES_REQUESTED** [review]
Almost! Just a couple of changes requested - see my comments. Here's the [Django docs](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#count) on `count()`

File: bangazonapi/views/lineitem.py (hunk 1 above)
Comment (jaclynmariefrench) [reply bangazonapi/views/lineitem.py]: Ok!! Good to know!!

File: bangazonapi/views/profile.py (hunk 2 above)
Comment (jaclynmariefrench) [reply bangazonapi/views/profile.py:188]: Got it!

**Review by heymonicakay — APPROVED** [review]

**Pull request merged**